doc/plutus-core-spec/notation.tex

```diff
@@ -114,13 +114,14 @@ \subsection{Bytestrings and bitstrings}
 \item We typically index the bytes in bytestrings starting from the
   \textit{left} end but the bits in bitstrings from the \textit{right} end.
 
-\item The bytestring $[c_0, \ldots, c_n]$ may be denoted by $c_0{\cdots}c_n$ ($n
-  \geq 1$, $c_i \in \B$);  the empty bytestring may be denoted by $\epsilon$.
-
-\item The bitstring $[b_n, \ldots, b_0]$ may be denoted by $b_n{\cdots}b_0$ ($n
-  \geq 1$, $b_i \in \b$); the empty bitstring may be denoted by $\epsilon$: we
-  also use this symbol for the empty bytestring, but this should not cause any
-  confusion.
+\item The bytestring $[c_0, \ldots, c_{n-1}]$ may be denoted by
```

Comment:
It's maybe a bit inconsistent to use zero-based indexing here, but it makes things a lot easier when dealing with the bitwise operations.  Note also that it was already like this: the only thing that's changed is the formatting.